README.md

```diff
@@ -1,2 +1,14 @@
 # siite
-The Quodl website - where the latest information on Quodl, the science behind it and sign up links can be found.
+The Quodl website - where the latest information on Quodl, the science behind it
+and sign up links can be found.
+
+## Set up
+- Clone this repo onto your machine
+- Check you have MongoDB, otherwise install it
+- Run `npm i` in your command line
+- Add the environment variable to connect with our hosted database to your `.env`
+file. Ask us and we will provide this for you.
```

Comment:
Maybe there should be a way to connect the local app with a database which is local as well as being able to connect to the production db.

Reply:
We do have this option in the code if the production db is not added, it's just that when people open the app it will be blank and they may be confused... how do you think that should be summarised here?

Reply:
I think there should be instructions for how to set up locally maybe with some default data (which will always work).